general_configuration/_topics/configuration.md

```diff
@@ -4,151 +4,96 @@
+The following diagram demonstrates an example of the multiple regions working together in a {{ site.data.product.title_short }} environment.
 
 ![7151](../images/7151.png)
 
-The Master appliance is located in Chicago and contains a master region
-and a subregion that manages the worker appliances. The Mahwah
-technology center contains a single subregion that manages two zones.
-Likewise the San Diego technology center contains a single subregion
-managing a single zone.
+The Master appliance is located in Chicago and contains a master region and a subregion that manages the worker appliances. The Mahwah technology center contains a single subregion that manages two zones.
+Likewise the San Diego technology center contains a single subregion managing a single zone.
 
 **Note:**
 
-  - Replicating a parent region to a higher-level parent is not
-    supported.
+  - Replicating a parent region to a higher-level parent is not supported.
 
   - Parent regions can be configured after the child regions are online.
 
 The following diagram provides a closer look at a region:
 ![7150](../images/7150.png)
 
-In this region, we have several {{ site.data.product.title_short }} appliances acting as UI
-nodes and worker nodes. These worker nodes execute tasks on the
-providers in your environment. The Region also uses a region database
-that reports to a master database on the main {{ site.data.product.title_short }} appliance.
-All appliances can connect to the authentication services (Active
-Directory, LDAP, Identity Management), outgoing mail (SMTP), and network
-services (SNMP).
+In this region, we have several {{ site.data.product.title_short }} appliances acting as UI nodes and worker nodes. These worker nodes execute tasks on the providers in your environment. The Region also uses a region database that reports to a master database on the main {{ site.data.product.title_short }} appliance.
+All appliances can connect to the authentication services (Active Directory, LDAP, Identity Management), outgoing mail (SMTP), and network services (SNMP).
 
 ##### Region Scope
 
-Regions are used to consolidate data from multiple VMDBs to a central
-database. The database at the top level, the master VMDB, cannot be used
-for operational tasks such as SmartState Analysis or Capacity and
-Utilization data collection. It is intended for use as a reporting
-database that includes all information across multiple subordinate
-regions. The subordinate regions replicate their information to the
-master.
+Regions are used to consolidate data from multiple VMDBs to a central database. The database at the top level, the master VMDB, cannot be used for operational tasks such as SmartState Analysis or Capacity and Utilization data collection. It is intended for use as a reporting
+database that includes all information across multiple subordinate regions. The subordinate regions replicate their information to the master.
```

Comment:
Not for this PR, but the terminology we like to use for multiple regions is "global region" and "remote region", instead of master and subordinate, respectively, in order to avoid that terminology.